### DISCUSSION TITLE
blog/2024-10-13-adding-comments-broke-my-site/

### DISCUSSION THREAD

**Discussion by giscus** [rnddave/oneMoreDavid-comments#3] [Announcements]
# blog/2024-10-13-adding-comments-broke-my-site/

Gatsby sucks, kinda. And what I'm doing about it

https://onemoredavid.com/blog/2024-10-13-adding-comments-broke-my-site/

**Comment by rnddave** [comment]
I forgot to test comments...

**Reply by rnddave** [reply]
Seem to be working
